src/Netatmo/AirClient.cs

```diff
@@ -0,0 +1,30 @@
+using System.IO;
+using System.Threading.Tasks;
+using Flurl.Http;
+using Netatmo.Models.Client;
+using Netatmo.Models.Client.Air;
+
+namespace Netatmo
+{
+    public class AirClient : IAirClient
+    {
+        private readonly string baseUrl;
+        private readonly ICredentialManager credentialManager;
+
+        public AirClient(string baseUrl, ICredentialManager credentialManager)
+        {
+            this.baseUrl = baseUrl;
+            this.credentialManager = credentialManager;
+        }
+
+        public Task<DataResponse<GetHomeCoachsData>> GetHomeCoachsData()
```

Comment:
Can you add `device_id` entry parameter like we do in WeatherClient ?
```suggestion
        public Task<DataResponse<GetHomeCoachsData>> GetHomeCoachsData(string deviceId = null)
```